src/engine/specs/CoreSpecs.ts

```diff
@@ -90,6 +91,16 @@ export interface IWallSpec extends IBaseSimObjectSpec {
   height?: number;
 }
 
+/**
+ * Spec for a Perimeter
+ */
+export interface IPerimeterSpec extends IBaseSimObjectSpec {
```

Comment:
This can just be a standalone interface, since the `IWallSpec` itself provides positional information